README.rst

```diff
@@ -30,6 +30,23 @@ configurations.
 .. _Cartographer: https://github.com/cartographer-project/cartographer
 .. _SLAM: https://en.wikipedia.org/wiki/Simultaneous_localization_and_mapping
 
+A Note for ROS Users
+====================
+
+**Cartographer is no longer actively maintained.**
```

Comment:
Is this note really only for ROS users or for everyone? The repository seems unmaintained completly.